workspaces/sonarqube/plugins/sonarqube-backend/README.md

```diff
@@ -16,14 +23,7 @@ In your `packages/backend/src/index.ts` make the following changes:
   backend.start();
 ```
 
-## Integrating into a backstage instance
-
-This plugin needs to be added to an existing backstage instance.
-
-```bash
-# From your Backstage root directory
-yarn --cwd packages/backend add @backstage-community/plugin-sonarqube-backend
-```
+## Old Backend System
```

Comment:
Note to self: we should remove support for the legacy backend as it's been deprecated in this plugin, not needed to be done thin this PR though.